freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/common/Status.java

```diff
@@ -97,4 +101,13 @@ public Boolean isStopInProgressPhase() {
     public Boolean isStoppedPhase() {
         return FREEIPA_STOPPED_STATUSES.contains(this);
     }
+
+    public boolean isCcmUpgradeablePhase() {
+        return FREEIPA_CCM_UPGRADEABLE_STATUSES.contains(this);
+    }
+
+    public boolean isCcmUpgradeInProgressPhase() {
```

Comment:
are you planning to use this? because currently it's not used so it shouldn't be here

Reply:
As of now not, so I removed